### ISSUE TITLE
Options to suppress certain updates

### ISSUE THREAD

**Issue by bryevdv** [python-versioneer/python-versioneer#247]
We always have to manually remove versioneer's edits to these files:
```
	modified:   MANIFEST.in
	modified:   bokeh/__init__.py
```
The change to `MANIFEST.in` is simply unnesessary/superfluous with our configuration. The `__init__.py` is actively interfere. We have additional versioning utilities and hooks on top of `versioneer` to accommodate the extra tests we need to do (we need to simulate different versions at test time) and we handle all that (including getting versioneer's version) in a `util.versions` module, not in `__init__.py` 

It would be nice to have some command line options to `versioneer install` to skip these modifications entirely.

**Comment by yarikoptic** [comment]
ran into the same desire for `__init__.py` since for API preservation purpose we would like to have injected code in `datalad/version.py` for now.  So it felt worthwhile having some `initfile_{source,build}` config option to override.

**Comment by effigies** [comment]
Yup, I run into this as well. What would be the preferred solution? Additional option in `setup.cfg` or a CLI flag like `--skip-snippet`? Something else?

**Comment by yarikoptic** [comment]
Config. I suggested one above for init to follow current config approach.
CLI options would cause inconsistencies across runs.

**Comment by effigies** [comment]
Ah. I can't read.
